### DISCUSSION TITLE
Recola's Plans are Set!

### DISCUSSION THREAD

**Discussion by hermonochy** [hermonochy/Rescueland#61] [Announcements]
Recola has now secured their plans for a calculated invasion. It has been disclosed that they intend to infiltrate Helucky to manipulate and control its forces. Once established, Recola plans to utilize Helucky's sheer military might to launch an assault against Rescueland with the objective of annihilating them.
This aggressive strategy aims to eliminate any resistance and ultimately secure Recola's dominance over the galaxy, all while maintaining a hidden presence in the background.

**Comment by badguyland** [comment]
What if Rescueland received leaked information about Recola's sinister plan? Is it possible that Recola takes too much risk?

**Reply by Klesone** [reply]
Maybe it doesn't matter, as Helucky forces are too great for Rescueland?

**Reply by hermonochy** [reply]
Yes, the Helucky forces considerably outpower those of Rescueland, even if Rescueland were given the time to prepare. However, the primary reason this information remains secure is that it is known solely within the innermost circle of Astan. This group is highly selective and operates with stringent security measures, ensuring that their discussions and plans are safeguarded from external scrutiny. Additionally, Recola's specific location remains undisclosed, shrouded in secrecy and adding another layer of complexity to their strategy. This combination of strong internal security and obscure geographical positioning creates a nearly impenetrable barrier against intelligence gathering by rival forces, helping Recola maintain the secrecy in their plans.

**Comment by Klesone** [comment]
Now that Recola has lost the Keluha base, they shall remain dormant until they can execute their master plan (see above).
